Second update User Guide - Add support for HTML response for WFS layer (#10582)
geosolutions-it/MapStore2: docs/user-guide/layer-settings.md

!!!note
    Without selecting any format here, the [Identify Tool](navigation-toolbar.md#identify-tool) will return the layers information with the format chosen in Map Settings ( in the [Side Toolbar](mapstore-toolbars.md#side-toolbar)). Once a user specifies the information format in layers settings, instead, that format will take precedence over the map settings only for that specific layer.

!!!note

>     MapStore allows users to choose the *Information format* for `WMS`, `CSW`, and `WFS` services.

Reviewer: ```suggestion
    From the Layer Settings panel, MapStore allows users to choose the *Information format* for `WMS` and `WFS` layer types.
```

>     The **Text** option is only available for `WMS` layers, while the **HTML** option for `WFS` layers is available only with GeoServer and if the [`wfs-freemarker`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) extension is installed on the GeoServer side. 

Reviewer 2: ```suggestion
    The **Text** option is only available for `WMS` layers, while the **HTML** option for `WFS` layers is available only with GeoServer and if the [`wfs-freemarker`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) extension is installed on the GeoServer side.
```

>     The **Text** option is only available for `WMS` and `CSW` services, while the **HTML** option for a `WFS service` is only available if that option is enable on the GeoServer side. A GeoServer can expose this kind of output when the [`wfs-freemarker-plugin`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) is included in the deploy.

Reviewer: ```suggestion
    The **Text** option is only available for `WMS` layers, while the **HTML** option for `WFS` layers is available only with GeoServer and if the [`wfs-freemarker`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) extension is installed on the GeoServer side. 
```
